doc/guides/maintaining-V8.md

```diff
@@ -3,15 +3,14 @@
 ## Background
 
 V8 follows the Chromium release schedule. The support horizon for Chromium is
-very different from the support horizon that Node.js needs to provide to its
-users. As a result Node.js needs to support a version of V8 for quite a bit
-longer than what upstream needs to support. Since V8 doesn't have an LTS
-supported branch, there is no official process around how the V8 branches in
-Node.js are maintained.
-
-This document attempts to document the current processes and proposes a workflow
-for maintaining the V8 branches in Node.js LTS and Current releases and how the
-Node.js and V8 teams at Google can help.
+different compared to the support horizon for Node.js. As a result, Node.js
+needs to support a version of V8 longer than what upstream needs to support.
+V8 branches in Node.js lack of official maintaining process due to a missing
+LTS supported branch.
+
+This document attempts to outline the current maintaining processes, proposes
+a workflow for maintaining the V8 branches in both Node.js LTS and current releases, and
```

Comment:
Could you be so kind and wrap this paragraph at 80 characters?

Reply:
Could you specify how can I achieve this without losing information. Also, I think this paragraph can be removed since the sections discussed in the paragraph were already explicitly stated in the document.

Reply:
Sorry, I meant wrap every line in the paragraph at 80 characters. Otherwise, our linter will complain :)

```markdown
This document attempts to outline the current maintaining processes, proposes
a workflow for maintaining the V8 branches in both Node.js LTS and current
releases, and discusses how the Node.js and V8 teams at Google can help.
```

Reply:
Sure, I will fix this. Sorry for the confusion and thank you again.